tf2onnx/graph_helper.py

```diff
@@ -0,0 +1,75 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT license.
+
+"""
+tf2onnx.graph_helper - class to help building graph, such as helping to make complex node
+"""
+
+import numpy as np
+from tf2onnx import utils
+
+
+class GraphHelper(object):
```

Comment:
How about a more meaning full name - if it is building the graph maybe GraphBuilder ?

Reply:
done